src/update/orderbook_rewards.py

```diff
@@ -9,42 +11,71 @@
 from src.models import AccountingPeriod
 
 
-def dune_repr(df_row: Series) -> str:
-    """
-    This is the per row format for inserting entries in a table of VALUES
-    Specific to a COW Reward Item
-    """
-    return (
-        f"('{df_row['receiver'].lower()}', {df_row['num_trades']}, {df_row['amount']})"
-    )
+class RewardQuery(Enum):
```

Comment:
I was just trying to minimize the amount of duplicated code in the most pragmatic way here. If we go this route we are going to need to implement each of these methods twice in a slightly different way and also introduce an abstract class enforcing that each of those methods is implemented. If we do decide to add a third push to user generated view that is a completely different dataset, I would go this route but at this moment I am not sure it needs to be generalized - since this is a hacky solution anyway I hope it doesn't stick around for too long.

How does that sound to you?

Reply:
>How does that sound to you?

Sounds good!